modules/virt-about-live-migration.adoc

```diff
@@ -13,4 +13,4 @@ You can use live migration if the following conditions are met:
 
 * Shared storage with `ReadWriteMany` (RWX) access mode.
 * Sufficient RAM and network bandwidth.
-* If the virtual machine uses a host model CPU, the nodes must support the virtual machine's host model CPU.
\ No newline at end of file
+* If the virtual machine uses a host model CPU, the nodes must support the virtual machine's host model CPU. By default, live migration traffic is encrypted using Transport Layer Security (TLS).
```

Comment:
The note about encryption is not part of the prerequisite. Please move it out of the bullet list.

Reply:
Done. Thanks.